docs/installing.rst

```diff
@@ -148,13 +148,30 @@ The following virtualization providers are suggested for the respective operatin
     # Set libvirtd to start automatically on system boot
     $ sudo systemctl enable libvirtd
 
------------------------------------------------------
-3. Install the vagrant-service-manager Vagrant plugin
------------------------------------------------------
+----------------------------------
+3. Install additional dependencies
+----------------------------------
+
+For some operating systems, you might need to install additional dependencies before you install the Vagrant plugins.
+
+* Fedora 24
+
+  Run the following commands to install the additional dependencies::
+
+  $ sudo dnf install "Development Tools"
```

Comment:
This is not the correct command 
```
$ sudo dnf install "Development Tools"
Last metadata expiration check: 0:24:11 ago on Tue Aug  9 11:34:14 2016.
No package Development Tools available.
Error: Unable to find a match.
```
Instead please use `sudo dnf install @'Development Tools'`